java-frontend/src/main/java/org/sonar/java/bytecode/se/BytecodeEGWalker.java

```diff
@@ -172,8 +174,17 @@ private void handleBlockExit(ProgramPoint programPosition) {
         case IFLT:
         case IFGE:
         case IFGT:
-        case IFLE:
-
+        case IFLE: {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Extract this nested code block into a method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1199)